### PR TITLE
Additional yum repos need to be added

### PR DESCRIPTION
**Pull request by hessio**
Signed-off-by: Patrick Hession <patrick.hession@seagate.com>

-----
[View rendered README.md](https://github.com/hessio/cortx-hare/blob/patch-2/README.md)

### CODE REVIEW

**Comment by knekrasov** [comment]
Thank you @hessio for submitting this PR.

Can you please reword your commit message in the format as follows

```
Problem: <short definition of the problem that you're addressing in this commit>

Solution: <How exactly your commit addresses the problem>
```

Note that this rule is initially taken from ZeroMQ http://wiki.zeromq.org/docs:contributing#toc3 since we were inspired by their [PC3 process](http://hintjens.com/blog:23).

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -59,6 +59,88 @@ and health-checking mechanisms.
...
+  enabled=1
+  ```
+  * C7.8.2003 repo
+  ```sh
+  vi /etc/yum.repos.d/C7.8.2003.repo
+  ```
+  * Paste:
+  ```sh
+  # C7.8.2003
+  [C7.8.2003-base]
```
Comment (mssawant) [inline README.md:83]: Why do we need this repo specifically for hare? looks like a generic one, so ideally I think it should be mentioned as part of cortx README.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,88 @@ and health-checking mechanisms.
...
+  name=CentOS-7.8.2003 - Fasttrack
+  baseurl=http://linuxsoft.cern.ch/centos-vault/7.8.2003/fasttrack/$basearch/
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+  enabled=1
+  ```
+* Make sure lustre that "lustre-whamcloud-2.12.5" is not listed:
+  * List repos:
+  ```sh
+  yum list lustre-client-devel kmod-lustre-client
```
Comment (mssawant) [inline README.md:121]: This needs to go in cortx-motr documentation and hare must add a link to motr installation document instead.

Comment (hessio) [reply README.md:121]: Oh ok this makes more sense. I will do that instead. I realized the other repos are not required this was an error with VM and are not required for building HARE.

Comment (knekrasov) [reply README.md:121]: @hessio Does it mean that we can close this PR?

Comment (hessio) [reply README.md:121]: Yes, let's do that!

**Pull request closed without merging**